packages/ui/src/components/manage-navigation/index.tsx

```diff
@@ -115,6 +116,15 @@ export const ManageNavigation = ({
     updatePinnedItems(currentPinnedItems.filter(pinnedItem => pinnedItem.id !== item.id))
   }
 
+  const permanantlyPinnedItems = useMemo(
+    () => currentPinnedItems.filter(item => item.permanentlyPinned),
+    [currentPinnedItems]
+  )
+  const draggablePinnedItems = useMemo(
+    () => currentPinnedItems.filter(item => !item.permanentlyPinned),
+    [currentPinnedItems]
+  )
+
```

Comment:
seperating `permanantlyPinnedItems` and `draggablePinnedItems` and rendering them separately.